unit_test/main_tests/main_platform_test.py

```diff
@@ -113,7 +118,7 @@ def test_archs_platform_specific(platform, intercepted_build_args, monkeypatch):
 
 
 def test_archs_platform_native(platform, intercepted_build_args, monkeypatch):
-    monkeypatch.setattr(platform_module, 'machine', lambda: 'x86_64')
+    monkeypatch.setattr(platform_module, 'machine', lambda: 'AMD64' if platform == 'windows' else 'x86_64')
```

Comment:
I think this monkeypatch is already done in the `platform` fixture?

Reply:
Not when I first wrote it. 😳